### PR TITLE
Added CUDA capability to time_sequence_prediction example

### PR DESCRIPTION
**Pull request by drscotthawley**
As per the gist of @t-vi , I added some CUDA calls to (automatically) enable some users to execute this example more quickly.

### CODE REVIEW

File: time_sequence_prediction/train.py [hunk 1]
```diff
@@ -8,19 +8,27 @@
...
+        self.use_cuda = use_cuda
 
     def forward(self, input, future = 0):
         outputs = []
-        h_t = Variable(torch.zeros(input.size(0), 51).double(), requires_grad=False)
-        c_t = Variable(torch.zeros(input.size(0), 51).double(), requires_grad=False)
-        h_t2 = Variable(torch.zeros(input.size(0), 51).double(), requires_grad=False)
-        c_t2 = Variable(torch.zeros(input.size(0), 51).double(), requires_grad=False)
+        if (self.use_cuda):
+            h_t = Variable(torch.zeros(input.size(0), 51).cuda(), requires_grad=False)
```
Comment (t-vi) [inline time_sequence_prediction/train.py]: Hello,

I'd do ```h_t = Variable(input.new(input.size(0),51).zero_())``` and similar and not use `self.use_cuda`.

Best regards

Thomas

Comment (drscotthawley) [reply time_sequence_prediction/train.py]: Hmm.  When I try that I get "AttributeError: 'Variable' object has no attribute 'new'"

$ python train.py 
STEP:  0
Traceback (most recent call last):
  File "train.py", line 86, in <module>
    optimizer.step(closure)
  File "/opt/anaconda/envs/py35/lib/python3.5/site-packages/torch/optim/lbfgs.py", line 101, in step
    orig_loss = closure()
  File "train.py", line 81, in closure
    out = seq(input)
  File "/opt/anaconda/envs/py35/lib/python3.5/site-packages/torch/nn/modules/module.py", line 224, in __call__
    result = self.forward(*input, **kwargs)
  File "train.py", line 23, in forward
    h_t = Variable(input.new(input.size(0),51).zero_())
  File "/opt/anaconda/envs/py35/lib/python3.5/site-packages/torch/autograd/variable.py", line 65, in __getattr__
    return object.__getattribute__(self, name)
AttributeError: 'Variable' object has no attribute 'new'

Comment (soumith) [reply time_sequence_prediction/train.py]: it should be `input.data.new`

Comment (drscotthawley) [reply time_sequence_prediction/train.py]: Thanks @soumith .  Added ".data" and updated as per @t-vi's comments.  Pull request will now show latest working version, with simplified Seq()

**Comment by msaroufim** [comment]
Now outdated

**Pull request closed without merging**